### PR TITLE
Update changelogs

### PR DESCRIPTION
**Pull request by xiangyan99**

### CODE REVIEW

File: sdk/core/azure-mgmt-core/CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 1.4.0 (Unreleased)
+## 1.4.0 (2023-04-06)
```
Comment (kashifkhan) [inline sdk/core/azure-mgmt-core/CHANGELOG.md:3]: just wanting to check, this is supposed to be empty ?

**Review by kashifkhan — APPROVED** [review]

File: sdk/core/azure-mgmt-core/CHANGELOG.md (hunk 1 above)
Comment (xiangyan99) [reply sdk/core/azure-mgmt-core/CHANGELOG.md:3]: Sorry. What is empty?

Comment (kashifkhan) [reply sdk/core/azure-mgmt-core/CHANGELOG.md:3]: no changes in the mgmt-core changelog ?

Comment (xiangyan99) [reply sdk/core/azure-mgmt-core/CHANGELOG.md:3]: The changes are already listed and non-needed bullets are already deleted. We only need to update release date. :)

**Pull request merged**